governance.md

```diff
@@ -0,0 +1,136 @@
+# Main Governance Document
+
+The Project
+===========
+
+The Jupyter/IPython Project (The Project) is an open source software project affiliated with the 501c3 NumFocus Foundation. The goal of The Project is to develop open source software and deploy open and public websites and services for reproducible, exploratory and interactive computing. The Software developed by The Project is released under the BSD (or similar) open source license, developed openly and hosted on public GitHub repositories under the ipython and jupyter GitHub organizations. Examples of Project Software include the IPython Notebook, the IPython Terminal, IPython.parallel, the Jupyter Hub, etc. The Services run by the Project consist of public websites and web-services that are hosted under the jupyter.org or ipython.org domains. The Services Examples of Project Services include the Jupyter and IPython websites ([http://jupyter.org](http://www.google.com/url?q=http%3A%2F%2Fjupyter.org&sa=D&sntz=1&usg=AFQjCNHztfVQq8SN6qni-DVgmoaTYBVasg) and [http://ipython.org](http://www.google.com/url?q=http%3A%2F%2Fipython.org&sa=D&sntz=1&usg=AFQjCNGiroA4bzjI-kCsgV4YdTPbgyuyeA)), nbviewer ([https://nbviewer.ipython.org](https://www.google.com/url?q=https%3A%2F%2Fnbviewer.ipython.org&sa=D&sntz=1&usg=AFQjCNGB-iNba6jJiu1N9Oo4oMTWwFq3Aw)) and the Jupyter coLaboratory ([https://colaboratory.jupyter.org](https://www.google.com/url?q=https%3A%2F%2Fcolaboratory.jupyter.org&sa=D&sntz=1&usg=AFQjCNFBxEodelaV8P877UGqxrp7WU8Y3g)).
```

Comment:
Hi all, thanks for the feedback. However, to have a full record of the
evolution of the document, I will like to merge it basically as is and then
start to fix the many things that need updates in further PRs.

On Wed, Sep 2, 2015 at 7:01 AM, Kyle Kelley <notifications@github.com>
wrote:

> In governance.md
> <https://github.com/jupyter/governance/pull/1#discussion_r38534589>:
>
> > @@ -0,0 +1,136 @@
> > +# Main Governance Document
> > +
> > +The Project
> > +===========
> > +
> > +The Jupyter/IPython Project (The Project) is an open source software project affiliated with the 501c3 NumFocus Foundation. The goal of The Project is to develop open source software and deploy open and public websites and services for reproducible, exploratory and interactive computing. The Software developed by The Project is released under the BSD (or similar) open source license, developed openly and hosted on public GitHub repositories under the ipython and jupyter GitHub organizations. Examples of Project Software include the IPython Notebook, the IPython Terminal, IPython.parallel, the Jupyter Hub, etc. The Services run by the Project consist of public websites and web-services that are hosted under the jupyter.org or ipython.org domains. The Services Examples of Project Services include the Jupyter and IPython websites ([http://jupyter.org](http://www.google.com/url?q=http%3A%2F%2Fjupyter.org&sa=D&sntz=1&usg=AFQjCNHztfVQq8SN6qni-DVgmoaTYBVasg) and [http://ipython.org](http://www.google.com/url?q=http%3A%2F%2Fipython.org&sa=D&sntz=1&usg=AFQjCNGiroA4bzjI-kCsgV4YdTPbgyuyeA)), nbviewer ([https://nbviewer.ipython.org](https://www.google.com/url?q=https%3A%2F%2Fnbviewer.ipython.org&sa=D&sntz=1&usg=AFQjCNGB-iNba6jJiu1N9Oo4oMTWwFq3Aw)) and the Jupyter coLaboratory ([https://colaboratory.jupyter.org](https://www.google.com/url?q=https%3A%2F%2Fcolaboratory.jupyter.org&sa=D&sntz=1&usg=AFQjCNFBxEodelaV8P877UGqxrp7WU8Y3g)).
>
> Looks like copying of links from a google search page.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/governance/pull/1/files#r38534589>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com